openslides/motions/config_variables.py

```diff
@@ -151,6 +151,19 @@ def get_config_variables():
         group='Motions',
         subgroup='Supporters')
 
+    # Comments
+
+    yield ConfigVariable(
+        name='motions_comments',
+        default_value='Comment',
+        input_type='text',
+        label='Comment fields for motions',
+        help_text='Use comma separated list of field names for public '
+                  'fields or use special JSON. See manual for syntax.',
```

Comment:
I prefere it to give an json example with internal and public fiels here.

Can you give me such a json example for testing? I tested with this code which does not work:
```
{{"name": "internal comment", "public": false},{"name": "public comment", "public": true}}
```